spec/features/merchants/show_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe 'Merchant show page', type: :feature do
+  context 'As a merchant user' do
+    describe 'When I visit my dashboard' do
+      before :each do
+        @merchant = create(:user, role: 1)
+        @item_1 = create(:item, user: @merchant)
+        @item_2 = create(:item, user: @merchant)
+        @item_3 = create(:item, user: @merchant)
+        @item_4 = create(:item, user: @merchant)
+        @user_1 = create(:user)
+        @user_2 = create(:user)
+        @user_3 = create(:user)
+        @order_1 = create(:order, user: @user_1, status: 1)
+        @order_2 = create(:order, user: @user_2, status: 1)
+        @order_3 = create(:order, user: @user_3, status: 0)
+        @order_4 = create(:order, user: @user_3, status: 0)
+        OrderItem.create!(item: @item_1, order: @order_1, quantity: 12, price: 1.99, fulfilled: true)
+        OrderItem.create!(item: @item_2, order: @order_2, quantity: 13, price: 1.99, fulfilled: true)
```

Comment:
I think this spec is failing for the same reason as below @zachlp2016 , just gotta mark these two OrderItems fulfilled:false